dodo.py

```diff
@@ -0,0 +1,11 @@
+import os
```

Comment:
Can you add a comment to the top line of each dodo.py file pointing to how it's used?